src/Container/Model/User/ChecksUniqueUsersEmailAddressFactory.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Prooph\ProophessorDo\Container\Model\User;
+
+use Interop\Container\ContainerInterface;
+use Prooph\ProophessorDo\Infrastructure\Service\ChecksUniqueUsersEmailAddressFromReadModel;
+use Prooph\ProophessorDo\Projection\User\UserFinder;
+
+/**
+ * Class ChecksUniqueUsersEmailAddressFactory
+ *
+ * @author Lucas Courot <lucas@courot.com>
+ */
+final class ChecksUniqueUsersEmailAddressFactory
+{
+    /**
+     * @param ContainerInterface $container
+     *
+     * @return ChecksUniqueUsersEmailAddressFromReadModel
```

Comment:
I prever an empty line between docblock annotation types, too. But this project seems to omitt the empty line, so you should omit it here also how in other files from that PR.